static/js/compose.js

```diff
@@ -1134,7 +1134,7 @@ $(function () {
             break;
         case 'FileTooLarge':
             // sanitization not needed as the file name is not potentially parsed as HTML, etc.
-            msg = "\"" + file.name + "\"" + i18n.t(" was too large; the maximum file size is 25MiB.");
+            msg = "\"" + file.name + "\" " + i18n.t("was too large; the maximum file size is 25MiB.");
```

Comment:
Can you also translate this string using a context. It would be make it more readable:
```JavaScript
context = {'file_name': file.name}
msg = i18n.t("\"__file_name__\" was too large; the maximum file size is 25MiB.", context);
```